clab/config/utils.go

```diff
@@ -101,35 +102,37 @@ func prepareLinkVars(lIdx int, link *types.Link, varsA, varsB Dict) error {
 		(varsB[vkFarEnd]).(Dict)[key] = v1
 	}
 
-	// Split all fields with a comma...
+	// Ensure values are added to both ends of the link
 	for k, v := range link.Vars {
-
-		r := SplitTrim(v)
-
 		if k == vkFarEnd || k == vkNodeName {
 			return fmt.Errorf("%s: reserved variable name '%s' found", link.String(), k)
 		}
 
-		if k == vkLinkIP && len(r) == 1 {
-			// calc the remote IP
-			ipF, err := ipFarEndS(v)
-			if err != nil {
-				return fmt.Errorf("%s: %s", link.String(), err)
+		vv := reflect.ValueOf(v)
+		if vv.Kind() == reflect.Slice || vv.Kind() == reflect.Array {
```

Comment:
can it possible be `Kind()==Array` here?

Reply:
Since the source of the data if from yaml, it could be safe just to use Array. If it was generated by anything else it it could be array-like / or a slice I guess.

Will try it

Reply:
Added a test case for it and Slice seems more preferred...